src/types.rs

```diff
@@ -247,7 +293,36 @@ impl From<LdkChannelDetails> for ChannelDetails {
 			is_channel_ready: value.is_channel_ready,
 			is_usable: value.is_usable,
 			is_public: value.is_public,
-			cltv_expiry_delta: value.config.and_then(|c| Some(c.cltv_expiry_delta)),
+			cltv_expiry_delta: value.config.map(|c| c.cltv_expiry_delta),
+			counterparty_unspendable_punishment_reserve: value
+				.counterparty
+				.unspendable_punishment_reserve,
+			counterparty_outbound_htlc_minimum_msat: value
+				.counterparty
+				.outbound_htlc_minimum_msat
+				.unwrap(), // This is only None if we serialized prior to LDK 0.0.109, which is not supported.
```

Comment:
felt that its a bit intimidating to have `unwrap` here so decided to add a comment